hail/python/hail/expr/type_parsing.py

```diff
@@ -14,9 +14,10 @@
     bool = "tbool" / "bool"
     call = "tcall" / "call"
     str = "tstr" / "str"
+    nat = ("tnat" / "nat") _ "<" identifier ">"
     locus = ("tlocus" / "locus") _ "<" identifier ">"
     array = ("tarray" / "array") _ "<" type ">"
-    ndarray = ("tndarray" / "ndarray") _ "<" type "," identifier ">"
+    ndarray = ("tndarray" / "ndarray") _ "<" type "," type ">"
```

Comment:
the type here should be `nat`, not `type`. Identifier was also wrong.

Reply:
`tvariable` is also an option, but I can restrict it to only those two.